pkg/setting/setting.go

```diff
@@ -145,6 +145,10 @@ var (
 	ImageUploadProvider string
 )
 
+type FeatureToggles interface {
+	IsEnabled(flag string) bool
+}
```

Comment:
Can you use the same interface defined in pkg/services/featuremgmt/models.go?

Reply:
I added better comments and made it deprecated -- this is only here to avoid circular dependency with the new helper function